src/style.css

```diff
@@ -0,0 +1,33 @@
+body {
+  margin: 0;
+}
+
+div {
```

Comment:
add general class name, avoid styling by tag name

Reply:
@irenhh HTML file wasn`t being changed because it was my task!
I can`t make changes in index.html !
Therefore, I can`t add a general class name, so I have decided to use just simple <div> tag